src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -274,6 +319,16 @@ function projective_space(::Type{NormalToricVariety}, d::Int; set_attributes::Bo
     pm_object = Polymake.fulton.NormalToricVariety(Oscar.pm_object(f))
     variety = NormalToricVariety(pm_object)
     
+    # make standard choice for weights of the cox ring
+    set_attribute!(variety, :torusinvariant_cartier_divisor_group, free_abelian_group(d+1))
+    set_attribute!(variety, :torusinvariant_weil_divisor_group, free_abelian_group(d+1))
+    set_attribute!(variety, :class_group, free_abelian_group(1))
+    set_attribute!(variety, :picard_group, free_abelian_group(1))
+    weights = matrix(ZZ,hcat([1 for i in 1:d+1]))
```

Comment:
Just FYI you could also do this:
```suggestion
    weights = matrix(ZZ,ones(Int,d+1,1))
```
It requires fewer allocations:
```
julia> @btime ones(Int,d+1,1);
  120.773 ns (1 allocation: 112 bytes)

julia> @btime hcat([1 for i in 1:d+1]);
  135.216 ns (4 allocations: 288 bytes)
```